### PR TITLE
Actions: DPI awareness to Win

### PR DESCRIPTION
**Pull request by fdennis**

### CODE REVIEW

File: .github/workflows/windows-x64.yml [hunk 1]
```diff
@@ -119,6 +119,12 @@ jobs:
         run: |
           choco install visualstudio2019community --package-parameters "--add Microsoft.VisualStudio.Component.VC.Redist.MSM"
 
+      - name: DPI awareness
+        shell: powershell
+        run: |
+          $env:PATH += ";C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x64"
+          mt.exe -manifest .\yubioath-desktop.exe.manifest -outputresource:.\release\yubioath-desktop.exe;#1
```
Comment (dagheyman) [inline .github/workflows/windows-x64.yml:126]: Should the `#1` in the end be there?

File: yubioath-desktop.exe.manifest [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (dagheyman) [inline yubioath-desktop.exe.manifest:1]: Can we put this file in resources/win ?

File: .github/workflows/windows-x64.yml (hunk 1 above)
Comment (fdennis) [reply .github/workflows/windows-x64.yml:126]: Yes, however, I don't think it's required.
https://docs.microsoft.com/en-us/windows/win32/sbscs/mt-exe

File: yubioath-desktop.exe.manifest (hunk 2 above)
Comment (fdennis) [reply yubioath-desktop.exe.manifest:1]: Makes sense, will move

File: .github/workflows/windows-x64.yml (hunk 1 above)
Comment (fdennis) [reply .github/workflows/windows-x64.yml]: Removed that part. Seems to work!

**Pull request merged**